paasta_tools/chronos_rerun.py

```diff
@@ -76,7 +76,8 @@ def modify_command_for_date(chronos_job, date):
         interpolate in the context of the date provided.
     """
     current_command = chronos_job['command']
-    chronos_job['command'] = chronos_tools.parse_time_variables(current_command, date)
+    if current_command is not None:
```

Comment:
I would expect that all the jobs have a command defined.
Checking the docs seems that `cmd`  is optional as the docker container could have an entry point defined.
Could we print something if command is None? I see that `paasta rerun` as `--verbose` option, we might write it there

Reply:
i agree we should warn user that something went wrong. personally i'd prefer to just fail here - how can we allow things to continue without command? something has gone terribly wrong.

Reply:
not necessarily @keymone  - command is optional (it falls back to using the default in the docker image).

sure, I can log an extra line if you think its helpful